tests/test_dss.py

```diff
@@ -542,6 +543,16 @@ def test_drip(self, mcd):
         assert mcd.pot.drip().transact()
 
 
+class TestOsm:
+    def test_price(self, web3, mcd):
```

Comment:
was thinking more like passing storage value to `_extract_price(storage_value)` but this is cool too :)